### ISSUE TITLE
[Screen Reader – Make Code - JavaScript] Screen reader doesn't announce the state after selecting 'Blocks & JavaScript' option present on 'Header' section.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3651]
**User Experience:**
If the correct  information and relationships are not provided, it would be difficult for people with different disabilities as the information and relationships that are implied by visual or auditory formatting will not be preserved when the presentation format changes

**Note:** User credentials should NOT be included in the bug. 

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Java Script or Python" control and select it.
6. Verify that state (Selected) of 'Blocks & JavaScript/Python' menu item controls by screen reader.

**Actual Result:**
Screen reader doesn't announce the state after selecting 'Blocks & JavaScript/Python' option present on 'Header' section.

**Expected Result:**
Screen reader should provide the state information  as 'Selected' after selecting 'Blocks & JavaScript/Python' option present on 'Header' section.

**Note:**
Same issue exists on Firefox + NVDA & Chrome + JAWS.

**WCAG Reference:**
[WCAG 1.3.1 - Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html})

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Screen Reader: Narrator, JAWS, NVDA

**Comment by Shalini3457** [comment]
Labels:
#Accessibility; #A11ySev2; #HCL; #Benchmark-HCL-MakeCode-OCT2020; #WCAG1.3.1; #Win10-Edge(Chromium)

[WCAG1.3.1_State information.zip](https://github.com/microsoft/pxt-microbit/files/5523051/WCAG1.3.1_State.information.zip)
